Demos

SergioCrisostomo opened the pull request:
Add demos to website.
(used Docs layout with ok from Pietro)

closes #162

SergioCrisostomo committed:
add demos builder

SergioCrisostomo committed:
add demos page

SergioCrisostomo committed:
add demos cache, middleware and POST echo

SergioCrisostomo committed:
add More to demos

SergioCrisostomo commented:
bump...

arian commented:
I'm getting a 404 for `/demos`.

arian commented:
More fundamentally, what do we want to achieve with the demos, are they still representable?

SergioCrisostomo committed:
fix path 404

SergioCrisostomo commented:
@arian sorry for the 404, fixed.

This PR fixes  broken links to demos and makes easy importing demos, from [the demos repo](https://github.com/mootools/mootools-demos) to the website.

If the demos are not representable (and I agree with you they need a update/upgrade) then we could/should ask the community to pimp them. 
We could also remove the ["Demos" header link in this PR](https://github.com/SergioCrisostomo/website/blob/demos/views/partials/header.jade#L39) so they are not present in the menu, still keeping them online and thus all the broken links working again.

arian commented:
I'd keep them out the main menu.

Some demos are a bit broken. For example:

- the position of the element in http://localhost:3000/demos?demo=Transitions
- http://localhost:3000/demos?demo=Element.Event.Pseudos and http://localhost:3000/demos?demo=Element.Event are swapped.

Also the HTML highlighting is wrong. Maybe because the containing element has `class="xml"`?

SergioCrisostomo committed:
hide from menu, fix highlight, fix demos order

 - hide “Demos” from menu
 - fix syntax highlight to detect css also
 - add colours for syntax highlight in css and xml/html tags
 - fix demo order that were being sorted alphabetically with
interference from file extension

SergioCrisostomo committed:
fix positioning inside main for demos

SergioCrisostomo commented:
@arian thanks for feedback! 
fixed and updated.

SergioCrisostomo commented:
@arian thanks for feedback! 

 Updated with:
- removed link from menu
- fixed highlight
- fixed broken demos

cocprobase commented:
I really enjoy simply reading all of your weblogs. Simply wanted to inform you that you have people like me who appreciate your work. Definitely a great post. clash of Clans Top 15 Best BH 5 Base Hats off to you Favorite Base Layouts for TH5
<a href="http://cocpro.org/">Clash of Clans Top 15 Best BH 5 Base</a>
